tests/ConstraintAnnotation.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Opstalent\CrudBundle\Tests;
+
+use Doctrine\Common\Annotations\Annotation;
+use Doctrine\Common\Annotations\DocParser;
+use PHPUnit\Framework\Constraint\Constraint;
+
+/**
+ * Class ConstraintAnnotation
+ * @author Szyon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+class ConstraintAnnotation extends Constraint
+{
+    /**
+     * @param mixed $class
+     * @return bool
+     */
+    public function matches($class)
+    {
+        $reflection = new \ReflectionClass($class);
+        $docParser = new DocParser();
+        $docParser->setIgnoreNotImportedAnnotations(true);
+        $docParser->setIgnoredAnnotationNames(['Annotation' => false]);
+        $docParser->setIgnoredAnnotationNamespaces([Annotation::class => false]);
+        $docParser->addNamespace(Annotation::class);
```

Comment:
`$this->getDocParser()` ? or `$this->prepareDocParser()`? it's not a logic of match

Reply:
fixed